TodoApp/TodoApp.Api.Tests/ItemsControllerTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
+using NUnit.Framework;
+using TodoApp.Api.Controllers;
+using TodoApp.Api.Models;
+using TodoApp.Api.Tests.Utils;
+
+namespace TodoApp.Api.Tests
+{
+    [TestFixture]
+    public class ItemsControllerTests
```

Comment:
Peter, I have discussed your solution with Petr and he got me some insight hot to think about dummy testing. Let's change your tests (again, sorry) a little bit.
You should copy the Items list from controller here (having the same copy is ok right now) and use them in the tests after all. But you should equal items you got from controller with the ones copied here.  Why? Because, when you will update the functionality of the controller, you will be forced to mock the services in the tests. Your tests will be ready for that by providing the mocking items array 😉.

If you have any question regarding this, please ask in person (me or Petr).